lib/lbt/bundle/BundleWriter.js

```diff
@@ -2,7 +2,8 @@
 
 
 const NL = "\n";
-const ENDS_WITH_NEW_LINE = /(^|\r\n|\r|\n)[ \t]*$/;
+const ENDS_WITH_NEW_LINE = /(\r\n|\r|\n)[ \t]*$/;
+const SPACES_OR_TABS_ONLY = /^[ \t]*$/;
```

Comment:
```suggestion
const SPACES_OR_TABS_ONLY = /^[ \t]+$/;
```

Otherwise this also matches empty strings which is not what the variable name says 🧐